debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresTaskContext.java

```diff
@@ -25,10 +28,15 @@
 @ThreadSafe
 public class PostgresTaskContext extends CdcSourceTaskContext {
 
+    protected final Logger logger = LoggerFactory.getLogger(getClass());
+
     private final PostgresConnectorConfig config;
     private final TopicSelector<TableId> topicSelector;
     private final PostgresSchema schema;
 
+    private Long lastXminTime;
```

Comment:
Can we make this an `Instant` or something similar? We're trying to converge to the more semantic JDK 8 temporal data types where possible.

Reply:
done